### PR TITLE
Add modal to display a single shoe product

### PR DESCRIPTION
**Pull request by Tolaniii7**
Add modal to display a single shoe product
Screenshot (17)

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -111,6 +111,7 @@ input {
 
 .button-two,
 .button-one {
+  /* margin: 0 auto; */
```
Comment (Wolemercy) [inline src/index.css]: Can we take this out?

File: src/pages/Cart/index.js [hunk 2]
```diff
@@ -69,18 +68,18 @@ function Carting({ product }) {
   return (
     <li className="cart-list">
       <div className="cart-item">
-        <img src={product.photo} alt={product.brandType} width="70" />
+        <img src={product.image} alt={product.name} width="70" />
         <div className="cart-details">
-          <p className="cart-brand">{product.brandType}</p>
-          <p className="cart-color">{product.colour}</p>
+          {/* <p className="cart-brand">{product.brandType}</p>
+          <p className="cart-color">{product.colour}</p> */}
```
Comment (Wolemercy) [inline src/pages/Cart/index.js]: Can we take these out please?

File: src/pages/Home/index.js [hunk 3]
```diff
@@ -72,46 +74,157 @@ function Deals({ item }) {
   );
 }
 
+function ProductModal({ selectedId, closeModal, product }) {
+  const [cart, setCarting] = useState([]);
```
Comment (Wolemercy) [inline src/pages/Home/index.js]:
```suggestion
  const [cart, setCart] = useState([]);
```

The state and the setter should be named the same

File: src/pages/Home/index.js [hunk 4]
```diff
@@ -72,46 +74,157 @@ function Deals({ item }) {
...
+  function handleAddProduct(product) {
+    setCarting((cart) => [...cart, product]);
+  }
+
+  function handleAdd() {
+    handleAddProduct(selectedId);
+  }
```
Comment (Wolemercy) [inline src/pages/Home/index.js]: Do we need to have these two separate functions? It seems like all `handleAdd` is doing is calling `handleAddProduct`. We should be able to make use of a single function for this. Perhaps try the following

```suggestion
  function handleAddProductToCart() {
    setCarting((cart) => [...cart, selectedId]);
  }
```

File: src/pages/Home/index.js [hunk 5]
```diff
@@ -72,46 +74,157 @@ function Deals({ item }) {
...
+
 function LatestProducts() {
+  const [product, setProduct] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [error, setError] = useState("");
+  const [selectedId, setSelectedId] = useState(null);
+  const [isOpen, setIsOpen] = useState(false);
+
+  function handleselectedProduct(product) {
+    // setSelectedId((selectedId) => (id === selectedId ? null : id));
```
Comment (Wolemercy) [inline src/pages/Home/index.js]: Can we take this out if we don't need it please?

File: src/pages/ProductDetails/index.js [hunk 6]
```diff
@@ -0,0 +1 @@
+
```
Comment (Wolemercy) [inline src/pages/ProductDetails/index.js]: Can we delete this file if it's not in use?

**Review by Wolemercy — CHANGES_REQUESTED** [review]
Looks good, but I made a few comments that require addressing

File: src/pages/Home/index.js (hunk 3 above)
Comment (Tolaniii7) [reply src/pages/Home/index.js]: Very true

File: src/pages/Home/index.js (hunk 4 above)
Comment (Tolaniii7) [reply src/pages/Home/index.js]: The  new code worked fine

**Review by Wolemercy — APPROVED** [review]
Fine work 🚀

**Pull request merged**